utils/spdx/build.gradle.kts

```diff
@@ -61,8 +61,10 @@ tasks.withType<AntlrTask>().configureEach {
     }
 }
 
-private val generateGrammarSource by tasks.existing
-sourceSets.main.get().java.srcDir(generateGrammarSource.map { files() })
+sourceSets.configureEach {
```

Comment:
commit message: How long are these job logs kept around? I wonder if it is better to inline the needed information into the commit message?!